Source/Editor/Content/Items/ContentItem.cs

```diff
@@ -353,11 +355,19 @@ public virtual void RefreshThumbnail()
         }
 
         /// <summary>
-        /// Updates the tooltip text text.
+        /// Updates the tooltip text.
         /// </summary>
         public virtual void UpdateTooltipText()
         {
-            TooltipText = "Path: " + Path;
+            string fileExtension = System.IO.Path.GetExtension(Path);
+            string fileDescription = Utilities.Utils.TranslateFileExtension(fileExtension);
```

Comment:
Actually, we could add `virtual` property getter for file description in `ContentItem` so each type would override it with proper info (eg. including custom assets made by game/plugin). Then `TranslateTypeName` could be implemented in `CSharpScriptItem`/`CppScriptItem`/etc. for each asset type - easier to extend. And I think that TranslateTypeNames` interface option can be removed as it is enough to have this filetype always visible (the extension is already in Path below).

Reply:
Yeah, I'm cool with that, BUT this solution is what I used because I don't want to go through every single type and try to make them all implement TranslateTypeName. Wouldn't that also break old stuff and stuff from other projects?

Reply:
Actually to add to this point, I'd like it if there were some way to only implement translatetypename on specific types, and for the other types it will fallback to the original thingy that I implemented, maybe that could be done through a base class. Actually reading your comment now, that might be what you mean lol

Reply:
Okay, no problem, I'll do it :)